src/store/metadata/oxigraph/store.py

```diff
@@ -258,7 +259,40 @@ def get_publishers(self) -> Optional[Dict]:  # pragma: no cover
         """
         Gets all publishers
         """
-        raise NotImplementedError
+
+        # Specify the named graph from which we are fetching data
+        graph = self.db
+        
+        # Use the construct wrappers to pull the raw RDF triples
+        # (as one rdflib.Graph() for each function) and add them
+        # together to create a sinlge Graph of the
+        # data we need.
+        result: Graph = (
+            construct_publishers(graph)
+        )
+
+        # Serialize the graph into jsonld
+        data = json.loads(result.serialize(format="json-ld"))
+
+        # Use a context file to shape our jsonld, removing long form references
+        data = jsonld.flatten(
+            data, {"@context": constants.CONTEXT, "@type": "hydra:Collection"}
+        )
+
+        # TODO Fix context weirdness - at the moment, the flatten() method is changing @type to `versions_url`
+        # also adds dcat: to the publishers
+        data["@graph"][0]["@type"] = "hydra:Collection"
+        data["@graph"][0]["publishers"] = data["@graph"][0].pop("dcat:publisher")
+
+        for idx, publisher in enumerate(data["@graph"][0]["publishers"]):
```

Comment:
Mike said try and avoid using `enumerate` if you can use a comprehension instead, e.g. this works as well:
```
    data["@graph"][0]["publishers"] = [
        self.get_publisher(x["@id"].split("/")[-1])
        for x in data["@graph"]
        if "landing_page" in x.keys()
    ]
    del data["@graph"][0]["dcat:publisher"]
```